interp/interpreter.go

```diff
@@ -928,12 +928,18 @@ func (r *runner) runAtRuntime(fn *function, inst instruction, locals []value, me
 		if inst.llvmInst.IsVolatile() {
 			result.SetVolatile(true)
 		}
+		if ordering := inst.llvmInst.Ordering(); ordering != llvm.AtomicOrderingNotAtomic {
+			result.SetOrdering(ordering)
```

Comment:
I think the volatile flag should also be set here (and similarly below for the store instruction).

Reply:
The volatile flag is already set for volatile loads/stores in the previous if statement. You can see this working in the changed part of `interp/testdata/revert.out.ll`.